.github/workflows/tests-rdbstorage.yml

```diff
@@ -0,0 +1,103 @@
+name: Tests (RDB storage)
+
+on:
+  push:
+    branches:
+      - master
+  pull_request: {}
+  schedule:
+    - cron: '0 23 * * SUN-THU'
+
+jobs:
+  tests-rdbstorage:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+
+    services:
+      mysql:
+        image: mysql:5.7
+        ports:
+          - 3306:3306
+        env:
+          MYSQL_ROOT_PASSWORD: mandatory_arguments
```

Comment:
Just curious but is this variable introduced for the same reason as what's discussed in https://github.community/t/mysql-service-never-comes-up-healthy-in-action/16890/10 ?

Reply:
You can find `MYSQL_ROOT_PASSWORD` is mandatory if you check [this page](https://hub.docker.com/_/mysql).